ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -126,9 +132,9 @@ private[apiserver] final class ApiTransactionService private (
       logger.info("Received request for transaction by event ID.")
     }
     logger.trace(s"Transaction by event ID request: $request")
-    ledger.EventId
+    LfEventId
```

Comment:
Just wondering: why have a renaming clause for the import of `EventId` just in this particular file?

Reply:
There's two `EventId`s which are incompatible, unfortunately.

Perhaps we'll rectify this at some point.